app/src/main/java/nl/nfi/cellscanner/BootDeviceReceiver.java

```diff
@@ -16,21 +16,24 @@
  */
 public class BootDeviceReceiver extends BroadcastReceiver {
 
+    private static final String TAG = BootDeviceReceiver.class.getSimpleName();
+
+
     @Override
     public void onReceive(Context context, Intent intent) {
 
         String action = intent.getAction();
-        String message = App.TITLE + " on boot action is " + action;
+        String message = CellScannerApp.TITLE + " on boot action is " + action;
         Toast.makeText(context, message, Toast.LENGTH_LONG).show();
 
         // TODO: @ check if the right application is sending the request
         /*
           Call the start of the application if the boot-up state of the application
           is inRecordingState
          */
-        if (Recorder.inRecordingState(context)){
-            if (PermissionSupport.hasAccessCourseLocationPermission(context)) Recorder.startService(context);
-            else Recorder.setRecordingState(context, false);
+        if (RecorderUtils.inRecordingState(context)){
+            if (PermissionSupport.hasAccessCourseLocationPermission(context)) RecorderUtils.startService(context);
+            else RecorderUtils.setRecordingState(context, false);
```

Comment:
Ik ben gewend de actie na de `if` of `else` op een nieuwe regel te beginnen met een indent (ongeveer zoals in python). Dacht dat dat wel gangbaar was, maar we hebben geen code standards voor java. Heeft Hansken zoiets?

Reply:
Zou ik eerlijk gezegt niet weten. Heb geen goed beeld van de Java standards, anders dan naamgeving. Bovenstaande is een gangbare manier wanneer je geen gebruik maakt van { } 
Dat toevoegen maakt het geheel mischien duidelijker